css/_responsive.scss

```diff
@@ -53,21 +53,6 @@
     }
 }
 
-@mixin full-size-modal-positioner() {
```

Comment:
Hey, joining the party a bit late, but @saghul asked me to also take a look. The point of these mixins was to ensure that the chat is rendered in a full-sized dialog past the 580px (on desktop) and 680px (on mobile, due to the larger buttons) thresholds. Not sure if this has been redone in any way, but from what I could see in a dev environment, it seems it hasn't. Please do let me know if I've tested this incorrectly, though :D

Attaching dev vs alpha.jitsi.net screenshots here:

<img width="564" alt="Screenshot 2021-01-18 at 09 47 05" src="https://user-images.githubusercontent.com/52234168/104886590-aee02100-5972-11eb-8d27-953dac426d22.png">

<img width="570" alt="Screenshot 2021-01-18 at 09 50 25" src="https://user-images.githubusercontent.com/52234168/104886601-b273a800-5972-11eb-8d08-a37f3d262dc4.png">

Reply:
Hi @muscat1 thanks for your feedback. Just to be sure, the expected behavior is to show the chat either attached to the left or in a fullscreen dialog but never like in the first screenshot attached above?

Reply:
That's correct, thanks for looking into it! It's already rendering inside a dialog at small widths, it just needs to be made fullscreen.